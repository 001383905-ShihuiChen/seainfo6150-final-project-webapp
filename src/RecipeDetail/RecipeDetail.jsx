import React, { useEffect, useState } from 'react';
import styles from './RecipeDetail.module.css';
const RecipeDetail = props => {
    return (
      <div className={styles.container}>
          <div className={styles.details}>
          <h2>See what people said</h2>
        <p >
            {props.shortText}
        </p>
          </div>
          <div className={styles.details}>
        
        <h2>Direction</h2>
        <p >
            {props.text}
        </p>
        </div>
      </div>
    );
  };
  
  export default RecipeDetail;