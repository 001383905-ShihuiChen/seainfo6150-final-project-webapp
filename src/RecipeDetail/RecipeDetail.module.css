.details {
    width: 500px;
    height: 400px;
    margin: 10px;
    border-radius: 15px;
    background-color: #b88282;
    text-align: center;
    padding: 25px;
    font-size: 20px;
    color: #301d11;
    font-family: 'Balsamiq Sans', cursive;
    box-shadow: 5px 5px 10px #ffcccc;
    border: 5px solid #984949;
  }
.container{
    margin-left: auto; 
    margin-right:auto; 
    display:block;
}

  