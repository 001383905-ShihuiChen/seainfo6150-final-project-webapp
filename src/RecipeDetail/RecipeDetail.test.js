import React from "react";
import { render } from "@testing-library/react";
import RecipeDetail from "./RecipeDetail.jsx";

it("renders the BookCoverImage component correctly", () => {
  const { container } = render(
    <RecipeDetail
        shortText = "This is great for a special brunch or even a quick and easy dinner. Other breakfast meats can be used, but the deli ham is the easiest since it is already fully cooked."
    />
  );
  expect(container).toMatchSnapshot();
});