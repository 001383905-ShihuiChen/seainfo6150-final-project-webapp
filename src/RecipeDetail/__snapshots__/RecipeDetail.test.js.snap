// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the BookCoverImage component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="details"
    >
      <h2>
        See what people said
      </h2>
      <p>
        This is great for a special brunch or even a quick and easy dinner. Other breakfast meats can be used, but the deli ham is the easiest since it is already fully cooked.
      </p>
    </div>
    <div
      class="details"
    >
      <h2>
        Direction
      </h2>
      <p />
    </div>
  </div>
</div>
`;
